examples/mega65/dma_audio.c

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 LLVM-MOS Project
+// Licensed under the Apache License, Version 2.0 with LLVM Exceptions.
+// See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
+// information.
+
+// MEGA65 DMA Audio example using C23
+
+#include <mega65.h>
+
+typedef unsigned _BitInt(24) uint24_t;
+
+// Sample data prepared with:
+// ~~~
+// sox -v 2.0 -V -D drums.wav drums.s8 remix - lowpass 7000 rate -v -s -I 11822 dither -S
+// ~~~
+static const uint8_t sample[] = {
+#embed "drums.s8"
```

Comment:
NOICE!

Reply:
yep, good timing for embed to arrive in llvm-mos :-)